grails-app/views/header/_mainPortalHeader.gsp

```diff
@@ -9,8 +9,8 @@
 <div id="header">
 
     <div id="logoContainer">
-        <a href="landing"><img src="${portalBranding.logoImage}" alt="logo" id="logo" />
-        </a>
+        <g:link controller="landing"><g:img uri="${resource(base: portalBranding.brandingBase, file: portalBranding.logoImage)}" alt="logo" id="logo" />
```

Comment:
I'm not reviewing this PR, but it seems like this logic of link generation should be in `portalBranding` if at all s.t. it returns a fully qualified link from `portalBranding` if needed.